crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -111,10 +123,19 @@ impl SizedBundle {
     }
 }
 
+pub(super) struct SizedSequenceAction(SequenceAction);
```

Comment:
why do we require an additional struct? i don't see any methods defined for it per say?

Reply:
`SizedBundle.push()` requires a `SizedSequenceAction`s rather than a regular `SequenceAction` to ensure that the `SequenceAction` went through `SizedBundle.push_check()` before being pushed.